spec/support/shared_examples/abilities/agency_super_examples.rb

```diff
@@ -8,4 +8,12 @@
   it "should be able to manage agency users" do
     should be_able_to(:manage_agency_users, User)
   end
+
```

Comment:
I don't think you need these ones in the `super` examples as well, since the ability spec runs the agency_with_refund tests for agency supers too.